packages/medusa-js/src/resources/admin/price-lists.ts

```diff
@@ -0,0 +1,83 @@
+import {
+  AdminPostPriceListPricesPricesReq,
+  AdminPostPriceListsPriceListPriceListReq,
+  AdminPostPriceListsPriceListReq,
+  AdminPriceListDeleteRes,
+  AdminPriceListRes,
+  AdminGetPriceListPaginationParams,
+  AdminPriceListsListRes,
+  AdminDeletePriceListPricesPricesReq,
+  AdminPriceListDeleteBatchRes,
+} from "@medusajs/medusa"
+import qs from "qs"
+import { ResponsePromise } from "../../typings"
+import BaseResource from "../base"
+
+class AdminPriceListResource extends BaseResource {
```

Comment:
Would be good to include `listProducts` as a method too. It will come with https://github.com/medusajs/medusa/pull/1239

This should be able to take both `id` and `query`